nevergrad/optimization/optimizerlib.py

```diff
@@ -977,6 +1016,21 @@ def _internal_tell_not_asked(self, candidate: p.Parameter, loss: tp.FloatLoss) -
             opt.tell(local_candidate, loss)
 
 
+class Rescaled(base.ConfiguredOptimizer):
+    """Configured optimizer for creating rescaled optimization algorithms.
+    """
+    # pylint: disable=unused-argument
+    def __init__(
+        self,
+        *,
+        base_optimizer: base.OptCls = CMA,
+        scale: tp.Optional[float] = None,
+    ) -> None:
+        super().__init__(_Rescaled, locals())
+
+
+RescaledCMA = Rescaled().set_name("RescaledCMA", register=True)
```

Comment:
should you remove Mini Milli Maxi whatever CMA as well?

Reply:
No. In the case of CMA there is already a method for that due to the special nature of the algorithm, which is equipped with a stepsize as many evolution strategies.